File: First_problem.cpp
/*In a tree are white globes, twice as many red globes and green globes as 3 less then the number of red globes.
How many globes are there?
Solution in C++:*/
#include<iostream>
using namespace std;
int main()
{
    long long white_globes, green_globes, red_globes,total=0;
    cin >> white_globes;
    red_globes = white_globes * 2;
    green_globes = red_globes - 3;
    white_globes = white_globes + red_globes + green_globes;
    cout << total;
}

File: First_problem.cs
/*In a tree are white globes, twice as many red globes and green globes as 3 less then the number of red globes.
How many globes are there?*/
using System;

namespace test1
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("How many globes are ?");
            int white_globes = Convert.ToInt32(Console.ReadLine());
            int red_globes = white_globes * 2;
            int green_globes = red_globes - 3;
            int total_globes = white_globes + red_globes + green_globes;
            Console.WriteLine("Total Globes:" + total_globes);
        }
    }
}
